platform/android/display_server_android.h

```diff
@@ -68,6 +68,13 @@ class DisplayServerAndroid : public DisplayServer {
 	bool control_mem = false;
 	bool meta_mem = false;
 
+	static const int ANDROID_MOUSE_WHEEL_UP = 1 << 6;
```

Comment:
There should be a comment here mentioning `GodotInputHandler.java` since the values need to stay in sync.